lib/plugins/xdrip-js.js

```diff
@@ -172,7 +172,7 @@ function init(ctx) {
         result.level = levels.WARN;
       }
 
-      if (sensorInfo.xdripjs.voltageb && (sensorInfo.xdripjs.voltageb < prefs.warnBatV)) {
+      if (sensorInfo.xdripjs.voltageb && (sensorInfo.xdripjs.voltageb < (prefs.warnBatV - 10))) {
```

Comment:
shouldn't the prefs.warnBatV not be lowered by 10 then?

Reply:
voltagea test uses the prefs.warnBatV value as is (not shown in the snippet above), but voltageb is offset to 10 less than the voltagea test value.